examples/io/elekta_average.py

```diff
@@ -0,0 +1,63 @@
+"""
```

Comment:
Can we add a `include_inactive` flag to the parser? Then could it be used with the `bst_phantom_elekta` example? That would both make it more functional and make this example nicer.